packages/core/src/external/opensearch/fhir-ingestor.ts

```diff
@@ -20,6 +20,9 @@ import { getEntryId } from "./shared/id";
 
 dayjs.extend(duration);
 
+const ERROR_CODES_TO_RETRY = [409, 503];
+const MAX_RETRY_ATTEMPTS = 3;
+const INITIAL_WAIT_TIME_BETWEEN_RETRIES = dayjs.duration({ seconds: 0.2 });
```

Comment:
There's the milliseconds unit let's use instead of decimal/partial numbers, please